### PR TITLE
fix: integrate new component-relay

### PR DESCRIPTION
**Pull request by johnny243**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
     <title>Folders</title>
+    <script src="dist/component-relay.js"></script>
```
Comment (atmoio) [inline index.html:6]: Shouldn't this be part of compiled.min.js?

Comment (johnny243) [reply index.html:6]: If I append the component relay to compiled.min.js, `ComponentRelay` is somehow `undefined`. Does not happen when building with `webpack` btw. We'll have to update dependencies on this repo in another PR.

Comment (atmoio) [reply index.html:6]: > If I append the component relay to compiled.min.js

You should prepend, not append.

Comment (johnny243) [reply index.html:6]: Yeah, I meant prepend. I replaced [this line](https://github.com/standardnotes/folders-component/blob/f423669c741e928a04bd6dfb0acfa3e2cebc1234/Gruntfile.js#L63) with `node_modules/@standardnotes/component-relay/dist/dist.js` so it's at the beginning of the compiled.min.js file. That gives me `ComponentRelay` is `undefined` error.

Comment (atmoio) [reply index.html:6]: I guess it's because they're using different module styles.

**Review by atmoio — APPROVED** [review]

**Pull request merged**